.github/workflows/jmeter.yml

```diff
@@ -37,6 +37,7 @@ jobs:
         run: |
           git clone https://github.com/lnbits/lnbits-extensions
           cd lnbits-extensions
+          git checkout pay_to_enable_extension_upgrades
```

Comment:
Before merging into dev:
 - remove this line
 - merge this PR: https://github.com/lnbits/lnbits-extensions/pull/360